BUILD.bazel

```diff
@@ -975,16 +975,24 @@ ray_cc_library(
     ],
 )
 
+ray_cc_library(
+    name = "raylet_client_connection_lib",
+    srcs = ["src/ray/raylet_client/raylet_connection.cc"],
+    hdrs = ["src/ray/raylet_client/raylet_connection.h"],
+    linkopts = select({
+        "@platforms//os:windows": [
+        ],
+        "//conditions:default": [
+            "-lpthread",
```

Comment:
Do we need the link option? i.e. if we remove the dependency, does it work for a cc_binary?
I never used it in my bazel build file

Reply:
Removed.